src/cephadm/cephadm.py

```diff
@@ -4678,6 +4698,40 @@ def probe_hba(scan_path: str) -> None:
 
     return f'Ok. {len(all_scan_files)} adapters detected: {len(scan_files)} rescanned, {len(skipped)} skipped, {len(failures)} failed ({elapsed:.2f}s)'
 
+def get_mgr_images() -> dict:
```

Comment:
It would be nicer to add this function somewhere other than cephadm.py to avoid growing it. But I wont require it.

Also, I worry about people not knowing about this function when adding new images to constants.py. Maybe not in this PR, but could we add some sort of more structured approach to the images.

Here's a hypothetical example:
```
class ContainerImage(typing.NamedTuple):
    name: str
    mgr_image_key: str

class DefaultImages(enum.Enum):
    PROMETHEUS = ContainerImage(
        'quay.io/prometheus/prometheus:v2.51.0',
        f'{_mgr_prefix}prometheus'
    )
```
Then in the future when people go to add new defaults to cephadm they see the pattern that the default images are named and that there's a matching key on the mgr.
Just a thought.

Reply:
Moved this function to container_types.py, Will create a ticket for having structured approach to images